### PR TITLE
Add Wikipedia clone 

### PR DESCRIPTION
**Pull request by amanver45**
Closes: #835 
Title: Add Wikipedia clone
- Name: Aman Verma
- Identify yourself:GSSOC Contributor

### Describe the add-ons or changes you've made 📃
ADD WIKIPEDIA CLONE 

### Screenshots 📷
LIGHT MODE Screenshot 2024-10-11 104021
DARK MODE Screenshot 2024-10-11 110939

- [x] My code follows the code style of this project.
- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by amanver45** [comment]
@Avdhesh-Varshney sir  check this pr i update the changes in both folder and readme

File: Vanilla-JS-Projects/Intermediate/Wikipedia-Clone/README.md
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Wikipedia-Clone/README.md:1]: README file is empty

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Vanilla-JS-Projects/Intermediate/Wikipedia-Clone/README.md
Comment (amanver45) [reply Vanilla-JS-Projects/Intermediate/Wikipedia-Clone/README.md:1]: update readme

**Comment by amanver45** [comment]
@Avdhesh-Varshney sir readme file is updated

**Comment by amanver45** [comment]
@Avdhesh-Varshney sir pls check it

**Comment by amanver45** [comment]
@Avdhesh-Varshney  sir check and merge it pls

**Comment by amanver45** [comment]
@Avdhesh-Varshney pls check it

**Comment by amanver45** [comment]
@Avdhesh-Varshney pls merge my pr

File: Vanilla-JS-Projects/Intermediate/Wikipedia-Clone/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+<!-- add the screenshot of the project (Mandatory) -->
+
+![image](screenshot.webp)
+
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h4 align='center'>Developed By <b><i>Heli Detroja</i></b></h4>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Wikipedia-Clone/README.md:53]: Is it you?

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Vanilla-JS-Projects/Intermediate/Wikipedia-Clone/README.md (hunk 1 above)
Comment (amanver45) [reply Vanilla-JS-Projects/Intermediate/Wikipedia-Clone/README.md]: sry sir readme is now updated

**Comment by amanver45** [comment]
@Avdhesh-Varshney readme is now updated pls check

**Comment by amanver45** [comment]
@Avdhesh-Varshney pls check my pr and merge it

**Comment by amanver45** [comment]
@Avdhesh-Varshney ..

**Comment by amanver45** [comment]
@Avdhesh-Varshney  check it sir

**Review by RadhikaMalpani1702 — COMMENTED** [review]
@MastanSayyad  review this PR

**Comment by MastanSayyad** [comment]
Hi @Avdhesh-Varshney Add only level 1 to all such PRs which has addition of projects instead of development on the repositories website

**Pull request merged**